### ISSUE TITLE
ICICI bank home page! radio button  is not in proper sequence 

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#15]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)
NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2.press H for heading till to reach to ‘I mobile -pay ’heading level 5and two times tab to reach tab ‘right product for you ‘
3. Reached to the tab ‘find the right product for you ‘
4.press tab  reached to combo box  and select ‘insurance’
5. press tab to reach next combo box and select ‘securing my loved one’
6.under the heading calculate your monthly premium press tab and then reached to the radio button
expected Result the radio button should in proper sequence 
Actual Result the screen reader announced five radio button but there is calendar placed after two radio button
WCAG Guideline
https://www.w3.org/TR/WCAG21/#meaningful-sequence

**Comment by manish10** [comment]
Hi Narendra,
This bug needs you to specify which radio button you are talking about. It is usually a good practice to mention the text label of the first radio button to identify the group.
It is not clear what is the problem with the sequence. It is important that someone reading the bug understands what is the problem and exactly what should be the actual result. It is not sufficient to say it should be in proper sequence. You need to tell what is the proper sequence.
